commentHandler.go

```diff
@@ -256,21 +263,78 @@ func manageLocking(req types.IssueCommentOuter, cmdType string) (string, error)
 	return buffer.String(), nil
 }
 
+func manageMilestone(req types.IssueCommentOuter, cmdType string, cmdValue string) (string, error) {
+
+	milestoneValue := cmdValue
+
```

Comment:
remove empty line.

Reply:
I always wonder what it means when I see `manage` in the name of some code, class or function. Usually it's the wrong name. If you still have this please can you re-consider an alternative?